flytekit/clients/friendly.py

```diff
@@ -51,6 +53,35 @@ def create_task(
             )
         )
 
+    def create_task_raw(
```

Comment:
Why do we need to duplicate this? if you want to use this, why not use the raw client directly?